### ISSUE TITLE
Can't install on node > 10

### ISSUE THREAD

**Issue by fletcherist** [jet2jet/js-synthesizer#4]
when running `npm i js-synthesizer --save` it throws the following
```bash
error js-synthesizer@1.5.2: The engine "node" is incompatible with this module. Expected version "10". Got "13.11.0"
```

could it be possible to allow not only node v10 but >10 too?
https://github.com/jet2jet/js-synthesizer/blob/master/package.json#L24

thanks

**Comment by fletcherist** [comment]
now i'm using `yarn install --ignore-engines` and it helps somehow

**Comment by jet2jet** [comment]
Thanks for the report.

I fixed this in #5, and will publish the new version of js-synthesizer in about 1 hour.

Edit: I published as v1.6.0.
